internal/handlers/pg/msg_createindexes.go

```diff
@@ -401,7 +402,9 @@ func processIndexOptions(indexDoc *types.Document) (*pgdb.Index, error) {
 				)
 			}
 
-			index.Unique = pointer.ToBool(true)
+			if unique {
+				index.Unique = pointer.ToBool(true)
+			}
```

Comment:
Are there any complications when we set `index.Unique` to false instead of nil?

```suggestion
			index.Unique = pointer.ToBool(unique)
```